cmds/exp/bootvars/main.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+// Command bootvars reads the current boot var. Given -m arg, it mounts the
```

Comment:
"the filesystem containing it"? 

That was confusing to me reading it for the first time. "Given -m, it mounts the filesystem pointed to by the current boot variable and prints its location."

Reply:
done